mappings/servers/kztserver/metadata.json

```diff
@@ -0,0 +1,37 @@
+{
+    "id": "kztserver",
+    "name": "KZTSERVER",
+    "description": "A light and fun Japanese server",
+    "addresses": [
+        "kztmc.net"
+    ],
+    "primaryAddress": "mc.kztmc.net",
+    "minecraftVersions": [
+        "1.8.9"
+    ],
+    "primaryMinecraftVersion": "1.8.9",
+    "primaryColor": "#ffaa00",
+    "secondaryColor": "#ffffff",
+    "primaryLanguage": "ja",
+    "languages": [
+        "en",
+        "ja"
+    ],
+    "primaryRegion": "JPN",
+    "regions": [
+        "JPN"

```

Comment:
That's not a valid region
Take a look at our [docs](https://docs.cheatbreaker.net/server-mappings/adding-servers/metadata#regions)